RELEASE_NOTES.md

```diff
@@ -2,6 +2,16 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 1.0.6 (May 9, 2024)
+
+- ♊ `shorebird release` and `shorebird patch` now accept `--platforms` to
+  specify which platforms to release or patch. This allows you to release or
+  patch multiple platforms in a single command. For example, `shorebird release
+  --platforms android,ios` will release both Android and iOS platforms.
```

Comment:
```suggestion
  --platforms android,ios` will release to both Android and iOS platforms.
```

Reply:
We also added support for `--dry-run` to both commands (previously just patch supported it)